Open Judge System/OJS.Common/Extensions/FileHelpers.cs

```diff
@@ -42,7 +44,7 @@ public static void UnzipFile(string fileToUnzip, string outputDirectory)
             }
         }
 
-        public static string FindFirstFileMatchingPattern(string workingDirectory, string pattern)
+        public static string FindFileMatchingPattern(string workingDirectory, string pattern)
```

Comment:
It's probably a good idea to add an optional delegate to the signature to specify the ordering, that way the method will be reusable, if you want to have a specific method for returning the biggest file, that should be a separate method.